src/DynamoRevit/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
```

Comment:
I probably would not commit this unless you think it's valuable for everyone?

Reply:
Seems generally useful to me...